### PR TITLE
Update LDAP-config-AD.md

### PR DESCRIPTION
**Pull request by rohit-sre-github**
Hey @mark-plummer 

1. We do not need the customer to obtain the AD SSL certificate from a third party or external authority. If customer has AD setup for SSL, it will already have its own SSL certificate installed and ThoughtSpot SREs can fetch/use that certificate using `openssl s_client -connect ad.yourdomain.local:636 > AD_SSL_certificicate.pem` and set it up in TS.
2. We don't support LDAP servers but only AD (Active directory). So replacing repeated references to LDAP by AD/Active Directory.
3. Provided meaningful example compliant with best practices (use domain name and not IP addresses).
4. Modified "4. If you are using SSL, [add the SSL certificate for LDAP](add-SSL-for-LDAP.html#)" to "4. If you are using SSL, [add the SSL certificate for AD]". Also, need to change "(add-SSL-for-LDAP.html#)" to "(add-SSL-for-AD.html#)"
5. Removed the reference asking the customer to contact support for using beta functionality "The Management Console". The idea behind the management console is to enable the customer to self-serve and we don't want to encourage customers to call up TS SRE team without even trying the console.

Please get this reviewed by Callosum team before merging.
Get in touch with me if any clarifications are needed.

### CODE REVIEW

**Comment by rohit-sre-github** [comment]
Even the name of the main page itself "Configure LDAP for Active Directory" should change to  "Configure authentication via Active Directory"

File: _admin/setup/LDAP-config-AD.md [hunk 1]
```diff
@@ -6,32 +6,22 @@ last_updated: tbd
...
-
-  **_NOTE:_** You can use this option only in `tscli`. This is useful in scenarios where some users are not in LDAP and are created only in ThoughtSpot.
-
+Before you configure ThoughtSpot for Active Directory, collect the following information:
+- **URL**: This is needed to connect to Active Directory. For example, `ldap://ad.yourdomain.local:389` or `ldap://ad.yourdomain.local:636`
+- **Domain name**: Default domain under which users who want to be authenticated against Active Directory reside. When a user logs in with a username, the default domain is added to the username before sending it to the LDAP server. If users reside in multiple domains, you can still designate one of them as the default. **_NOTE:_** Users belonging to a non-default domain will have to explicitly qualify their username when they log in, for example: `username@ad.yourdomain.local`
```
Comment (csurfer) [inline _admin/setup/LDAP-config-AD.md]: > If users reside in multiple domains, you can still...

This has to be multiple sub-domains. We do not support authenticating against multiple domains. We support authentication against multiple subdomains Ex: `na.thoughtspot.com`, `au.thoughtspot.com` etc but not `thoughtspot.com` and `ts.com`.

File: _admin/setup/LDAP-config-AD.md [hunk 2]
```diff
@@ -6,32 +6,22 @@ last_updated: tbd
...
+- **Search base**: LDAP search base (Scope of searching for user/information within AD).
```
Comment (csurfer) [inline _admin/setup/LDAP-config-AD.md]: > (Scope of searching for user/information within AD)

(Scope for searching user-information like email and displayName within AD)

File: _admin/setup/LDAP-config-AD.md [hunk 3]
```diff
@@ -6,32 +6,22 @@ last_updated: tbd
...
+- **Automatically add LDAP or AD users in ThoughtSpot? (yes/no)**: If you choose 'yes', new users in ThoughtSpot will be automatically created when authenticated against AD. Note that ThoughtSpot doesn't cache passwords for AD-authenticated users. If you choose 'no', users have to be manually created with a dummy password as a placeholder in ThoughtSpot before they can log in. The username you specify when creating the LDAP authenticated user manually in ThoughtSpot has to be domain qualified, for example: `username1@ad.yourdomain.local`. **_NOTE:_** In order to log in to ThoughtSpot, the user has to exist in ThoughtSpot independent of whether that user is authenticated against AD or against ThoughtSpot's internal authentication.
```
Comment (csurfer) [inline _admin/setup/LDAP-config-AD.md]: > If you choose 'yes', new users in ThoughtSpot will be automatically created when authenticated against AD.

If you choose 'yes', new users will be automatically created within ThoughtSpot when successfully authenticated against AD.

File: _admin/setup/LDAP-config-AD.md [hunk 4]
```diff
@@ -6,32 +6,22 @@ last_updated: tbd
...
 
 
 ## Before you begin
-Before you configure LDAP for Active Directory, collect the following information:
-- **URL**: This is needed to connect to Active Directory. For example, `ldap://192.168.2.48:389`
-- **Domain name**: Preferred domain under which users who want to be authenticated against Active Directory reside. When a user logs in with a username, the default domain is added to the username before sending it to the LDAP server. If users reside in multiple domains, you can still designate one of them as the default.
-
-  **_NOTE:_** Users belonging to a non-default domain will have to explicitly qualify their username when they log in, for example: `username@ldap1.thoughtspot.com`
-- **Search base**: LDAP search base that allows ThoughtSpot to find user properties such as email and display name from LDAP.
-- **SSL**: Obtain the SSL certificate from an issuing authority if you want to use SSL.
```
Comment (csurfer) [inline _admin/setup/LDAP-config-AD.md]: This information is useful to have but the wording seems wrong. Some other information to keep in mind are:

1. If AD servers are sitting behind a load balancer then the setup should be such that we procure the SSL certificate to identify ourself to the load balancer and the communication after LB is non-secure. 
2. We don't support scenario where there are multiple AD servers and each of them provides its own SSL cert.

**Pull request merged**